### PR TITLE
Add Italian translation

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+msgid "Exec File"
+msgstr File esecutivo""
+
+#: src/Views/EditView.vala:79
+msgid ""
+"The command/app launched when you click the app entry in the launcher. "
+"Specify in an absolute path or an app's alias name."
+msgstr ""
+"Il comando/l'app è avviata quando si fa clic sulla voce dell'app nel programma di avvio. "
+"Specificare in un percorso assoluto o il nome alias di un'app.
```
Comment (ryonakano) [inline po/it.po:61]:
```suggestion
"Specificare in un percorso assoluto o il nome alias di un'app."
```

Seems like you removed the trailing quotation mark by mistake.

**Review by ryonakano — CHANGES_REQUESTED** [review]
Hi, thanks as always! Leaving one request need to be changed.

File: po/it.po (hunk 1 above)
Comment (albanobattistella) [reply po/it.po]: HI,
Done!

おはようございます！

File: po/it.po [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+#: src/Views/FilesView.vala:23
+msgid "No valid app entries found"
+msgstr "Nessuna voce valida per l'app trovata"
+
+#: src/Views/FilesView.vala:24
+msgid ""
+"If you've never created one, go back to Welcome View and click “New Entry”."
+msgstr ""
+"Se non ne hai mai creato uno, torna alla visualizzazione di benvenuto e fai clic su "Nuova voce"."
```
Comment (ryonakano) [inline po/it.po:124]:
```suggestion
"Se non ne hai mai creato uno, torna alla visualizzazione di benvenuto e fai clic su “Nuova voce”."
```

The source string uses `”`, not `"`. A tiny thing but this causes the build error

**Review by ryonakano — CHANGES_REQUESTED** [review]

File: po/it.po [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+msgstr "Cancella…"
+
+#: src/Views/FilesView.vala:82
+msgid "Edit…"
+msgstr "Modifica…"
+
+#: src/Views/FilesView.vala:100 src/Views/FilesView.vala:113
+#, c-format
+msgid "Are you sure you want to delete “%s”?"
+msgstr "Sei sicuro di voler eliminare "%s"?"
```
Comment (ryonakano) [inline po/it.po:137]:
```suggestion
msgstr "Sei sicuro di voler eliminare “%s”?"
```

**Review by ryonakano — CHANGES_REQUESTED** [review]

**Review by ryonakano — APPROVED** [review]
Fixed all errors. Thank you for your translation!

**Comment by albanobattistella** [comment]
> Fixed all errors. Thank you for your translation!

thanks to you!

**Pull request merged**